example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
+enabled = true
+timeout = 5
```

Comment:
We should add two timeouts for ``last_bind`` and ``first_bind``. See #14. Maybe the naming also is not perfect, yet.